src/installation/install_epinio_manual.md

```diff
@@ -0,0 +1,120 @@
+# Install Epinio (the manual way)
+
+As described [in the top level document](src/installation/installation.md#installation-methods), this method is intended for production level setups where operators want to have full control of the process.
+
+## Prerequisites
+
+Epinio needs a number of external components to be running on your cluster in order to work. You may already have those deployed, otherwise follow the instructions here to deploy them.
+
+Important: Some of the namespaces of the components are hardcoded in the Epinio
+code and thus are important to be the same as described here. In the future this
+may be configurable on the Epinio Helm chart.
+
+### Linkerd
+
+- Optional
```

Comment:
If it is optional we should explain what it is useful for.

Reply:
Added a sentence